core/actions/assertion.ts

```diff
@@ -75,8 +75,6 @@ export class Assertion extends ActionBuilder<dataform.Assertion> {
       this.query(nativeRequire(config.filename).query);
     }
 
-    // TODO(ekrekr): load config proto column descriptors.
```

Comment:
Why remove this  comment in assertion?

Reply:
The comment is a lie - columns aren't a field on assertions https://cloud.google.com/dataform/docs/reference/dataform-core-reference#iassertionconfig